core/client/fs/src/main/java/alluxio/client/journal/JournalMasterClient.java

```diff
@@ -40,7 +43,18 @@
    * Changes the leading master of the quorum.
    *
    * @param newLeaderNetAddress server address of the new leader
+   * @return the old peer priorities before the transfer of leadership
    * @throws AlluxioStatusException
    */
-  void transferLeadership(NetAddress newLeaderNetAddress) throws AlluxioStatusException;
+  TransferLeadershipPResponse transferLeadership(NetAddress newLeaderNetAddress)
```

Comment:
Let's not mix priorities with transfer functionality at the moment. I believe `void` is fine.